src/operator/gitea.ts

```diff
@@ -137,6 +143,86 @@ const secretsAndConfigmapsCallback = async (e: any) => {
       break
   }
 }
+// Exported for testing purposes
+export const addServiceAccountToOrganizations = async (
+  organizationApi: OrganizationApi,
+  serviceAcountName: string,
+  organisations: Organization[],
+) => {
+  const organisation = organisations.find((org) => serviceAcountName === `organization-${org.name}`)
+  const teams: Team[] = await doApiCall(errors, `Getting teams from organization: ${organisation?.name}`, () =>
+    organizationApi.orgListTeams(organisation!.name!),
+  )
+  const ownerTeam = teams.find((team) => team.name === 'Owners')
+  const members: User[] = await doApiCall(errors, `Getting members from Owners team in ${organisation?.name}`, () =>
```

Comment:
It might be difficult to replace existing calls all at once, but should we avoid adding more instances of doApiCall? The so-called error handling currently consists of exiting with status 1, while the log in the meantime gets full with errors and their consequential errors. That is the opposite of useful, since the stacktrace is lost.